goodmap/db.py

```
@@ -104,6 +102,404 @@
 
 
+# ------------------------------------------------
+# add_location
+
+def json_file_db_add_location(self, location_data, location_model):
+    location = location_model.model_validate(location_data)
+    with open(self.data_file_path, "r") as file:
+        json_file = json.load(file)
+
+    map_data = json_file["map"].get("data", [])
+    idx = next((i for i, point in enumerate(map_data) if point.get("uuid") == location_data["uuid"]), None)
```

Ruff (0.11.9)

Line too long (107 > 100)

(E501)